youtube_dl/extractor/freesound.py

```diff
@@ -23,17 +31,52 @@ def _real_extract(self, url):
         mobj = re.match(self._VALID_URL, url)
         music_id = mobj.group('id')
         webpage = self._download_webpage(url, music_id)
-        title = self._html_search_regex(
-            r'<div id="single_sample_header">.*?<a href="#">(.+?)</a>',
-            webpage, 'music title', flags=re.DOTALL)
         description = self._html_search_regex(
             r'<div id="sound_description">(.*?)</div>', webpage, 'description',
             fatal=False, flags=re.DOTALL)
 
+        duration = float_or_none(get_element_by_class('duration', webpage))
+        if duration:
+            duration = duration / 1000
+
+        tags = get_element_by_class('tags', webpage)
+        sound_info = get_element_by_id('sound_information_box', webpage)
+        release_date = get_element_by_id('sound_date', webpage)
+        filesize = float_or_none(self._search_regex(
+            r'Filesize</dt><dd>(\d+.\d+).*</dd>', sound_info, 'file size (approx)', fatal=False))
+        if filesize:
+            filesize = filesize * 1048576
```

Comment:
`parse_filesize()` shows a bit smaller size (MB * 1000 instead of MB * 1024), but I don't think it's a big of an issue because it shows filesize approximately anyway.